examples/sample.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+
+from __future__ import print_function
+import uharfbuzz as hb
+import sys
+
+
+with open(sys.argv[1], 'rb') as fp:
+    fontdata = fp.read()
+
+text = sys.argv[2]
+
+face = hb.Face.create(fontdata, 0)
+font = hb.Font.create(face)
+upem = face.upem
+del face
```

Comment:
nit: font keeps a reference to face, so "del face" isn't doing much here. Is it here to somehow highlight that face isn't used after that point?

Reply:
not really, was just a blind copy from harfbuzz own example script